Makefile

```diff
@@ -7,5 +7,10 @@ publish-ci-image:
 	docker buildx build --push --pull --platform linux/amd64 -t docker/tilt-extensions-ci -f .circleci/Dockerfile .circleci
 
 shellcheck:
-	find . -type f -name '*.sh' -not -path "*/node_modules/*" -not -path "*/.git-sources/*" -not -path "*/.git/*" -exec \
-	    docker run --rm -it -e SHELLCHECK_OPTS="-e SC2001" -v $$(pwd):/mnt nlknguyen/alpine-shellcheck {} \;
+	LINT_ERRORS=0; \
+	IFS=$$'\n'; for shellscript in $$(git ls-files -z | xargs -0 file | grep "shell script" | cut -d: -f1 || echo ""); do \
```

Comment:
rather than adding more complex shell scripting in the makefile, can you create a shell script in a `hack` directory? that's usually the kubernetes convention

Reply:
removed the complex command. will explain in another comment why/how.